src/Promenade.Service/LocationService.cs

```diff
@@ -271,7 +271,7 @@ private string GetFormattedDayGroupings(List<DayOfWeek> days)
 
                 if (days.Count == lastDay - firstDay + 1)
                 {
-                    return $"{dayFormatter.GetAbbreviatedDayName(firstDay)} \u2014 {dayFormatter.GetAbbreviatedDayName(lastDay)}";
+                    return $"{dayFormatter.GetAbbreviatedDayName(firstDay)} \u2015 {dayFormatter.GetAbbreviatedDayName(lastDay)}";
```

Comment:
Use an entity name rather than a unicode glyph here. Probably `&ndash;` for a date range, no spaces on either side of the dash.